.pipelines/templates/template-job-deploy-azure-env-tag.yml

```diff
@@ -0,0 +1,119 @@
+# Azure DevOps Job deploying rp
+parameters:
+  aroVersionStorageAccount: ''
+  azureDevOpsE2EJSONSPN: ''
+  azureDevOpsJSONSPN: ''
+  billingE2EPipelineName: ''
+  billingE2EBranchName: ''
+  configVersion: ''
```

Comment:
do we need this here if we're not using it then?

Reply:
damn, as I said it was late that day already. Good finding. I have removed it. Thanks :)